modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -160,15 +170,69 @@ spec:
   baseDomain: example.openshift.com
   privateZone: <1>
     id: mycluster-100419-private-zone
+ifndef::user-infra-vpc[]
   publicZone: <1>
     id: example.openshift.com
+endif::user-infra-vpc[]
 status: {}
 ----
-<1> Remove these sections completely.
+<1> Remove this section completely.
 +
+ifndef::user-infra-vpc[]
 If you do so, you must add ingress DNS records manually in a later step.
+endif::user-infra-vpc[]
 endif::gcp,aws,azure[]
 
+ifdef::user-infra-vpc[]
+. Configure the cloud provider for your VPC.
++
+--
+.. Open the `<installation_directory>/manifests/cloud-provider-config.yaml` file.
+.. Add the `network-project-id` parameter and set its value to the name of project that hoststhe shared VPC.
+.. Add the `network-name` parameter and set its value to the name of the network that hosts the shared VPC.
+.. Replace the value of the `subnetwork-name` parameter with the value of the subnet on the shared VPC to host your compute machines.
++
+--
+The contents of the `<installation_directory>/manifests/cloud-provider-config.yaml` resemble the following example:
++
+----
+config: |+
+  [global]
+  project-id      = example-project
+  regional        = true
+  multizone       = true
+  node-tags       = opensh-ptzzx-master
+  node-tags       = opensh-ptzzx-worker
+  node-instance-prefix = opensh-ptzzx
+  external-instance-groups-prefix = opensh-ptzzx
+  network-project-id = example-shared-vpc
+  network-name    = example-network
+  subnetwork-name = example-worker-subnet
+----
+
+. Open the `<installation_directory>/manifests/cluster-ingress-default-ingresscontroller.yaml` file and replace the value of the `scope` parameter with `External`. The contents of the file resemble the following example:
```

Comment:
This procedure is only required by external cluster. Private cluster should ignore it.